geps/gep-1897.md

```diff
@@ -0,0 +1,113 @@
+# GEP-1897: TLS from Gateway to Backend for ingress
+
+* Issue: [#1897](https://github.com/kubernetes-sigs/gateway-api/issues/1897)
+* Status: Provisional
+
+## TLDR
+
+This document specifically addresses the topic of conveying HTTPS from the Gateway
+dataplane to the backend (backend TLS termination), and intends to satisfy the single
+use case “As a client implementation of Gateway API, I need to know how to connect to
+a backend pod that has its own certificate”. TLS configuration can be a nebulous topic,
+so in order to drive resolution this GEP focuses only on this single piece of functionality.
+
+## Goals
+
+1. The solution should satisfy the following use case: the backend pod has its own
```

Comment:
We should mention the approach to the x-ns reference between Route and Service.

As mentioned on the call -- it would help to say explicitly that we are going to use the standard rules wrt to direct reference configuration.

Reply:
Thanks @bowei, I will be sure to mention this in the implementation section, as I think the description will be dependent on the implmentation details.

Reply:
Agree that a `TODO` and/or Unresolved section about this so we don't lose track of it. Let me know if this already happened and I missed it.

Reply:
I will add an Open Questions section and try to record anything having to do with implementation details.

Reply:
Added to Open Questions.